readme.md

```diff
@@ -250,6 +250,39 @@ got.post('google.com', {
 ```
 
 
+## OAuth
+
+You can use the [`oauth-1.0a`](https://www.npmjs.com/package/oauth-1.0a) module to create a signed OAuth request:
```

Comment:
I copied that from the previous example. I am going to change form-data to link to github too.